test/features/bootstrapper/remote_image_download_csi.go

```diff
@@ -0,0 +1,89 @@
+//go:build e2e
+
+package bootstrapper
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta4/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta4/dynakube/oneagent"
+	"github.com/Dynatrace/dynatrace-operator/test/features/cloudnative/codemodules"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers"
+	dynakubeComponents "github.com/Dynatrace/dynatrace-operator/test/helpers/components/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/kubeobjects/job"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/kubeobjects/namespace"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/kubeobjects/pod"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/sample"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/tenant"
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+const (
+	bootstrapperImage = "quay.io/dynatrace/dynatrace-bootstrapper:snapshot"
+)
+
+func InstallWithCSI(t *testing.T) features.Feature {
+	builder := features.New("remote-image-download-with-csi")
```

Comment:
The feature has been renamed to "node-image-pull" recently (#4601). This should be reflected in the tests as well.

Reply:
yup, already talked to @0sewa0 and I am fixing currently, just needed to merge main to my branch.